gatsby/content/projects/sdks/simple-matrix-bot-lib.mdx

```diff
@@ -0,0 +1,63 @@
+---
+layout: project
+title: Simple-Matrix-Bot-Lib
+categories:
+  - sdk
+description: An easy to use bot library for the Matrix ecosystem written in Python.
+repo: https://github.com/KrazyKirby99999/simple-matrix-bot-lib
+featured: true
+language: Python
+author: KrazyKirby99999
+license: MIT
+home: https://simple-matrix-bot-lib.readthedocs.io/
+room: "#simplematrixbotlib:matrix.org"
+e2e: "no"
+thumbnail: /docs/projects/images/python.png
+screenshot: /docs/projects/images/python.png
+maturity: Released
+---
+
+Simple-Matrix-Bot-Lib is a Python bot library for the Matrix ecosystem built on matrix-nio.
```

Comment:
We should add a link here so it's easy for people to reference what is being talked about:

```suggestion
Simple-Matrix-Bot-Lib is a Python bot library for the Matrix ecosystem built on [matrix-nio](https://github.com/poljar/matrix-nio)
```